docs/concepts/Auditing-Packages.md

```diff
@@ -115,7 +115,44 @@ Alternatively, if you want to keep low and moderate vulnerabilities as warnings,
 > [!NOTE]
 > MSBuild properties for message severity such as `NoWarn` and `TreatWarningsAsErrors` are not supported for packages.config projects.
 
-## Ensure restore audited projects
+## Running NuGet Audit in CI
+
+### Warnings or errors
```

Comment:
feels like there should be a subheader for audit pipeline or something.

Reply:
What do you all think of "Treat audit warnings as errors in dedicated pipeline"?

"wielding errors" doesn't mean anything to me 😆

Reply:
My brain skipped the wielding part lol. 
That's why I said a variation. 

Dedicated auditing pipeline?

Reply:
"Separating Errors from Warnings with a Dedicated Auditing Pipeline"